### PR TITLE
Chore: Artillery target port fixed

### PR DESCRIPTION
**Pull request by MarMarMot**

### CODE REVIEW

**Review by Kammerlo — APPROVED** [review]

File: artilleryTests/rosetta-load-test.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 config:
-  target: http://localhost:8081
+  target: http://127.0.0.1:8082
+  ensure:
+    maxErrorRate: 1
   http:
     defaults:
```
Comment (BerezinD) [inline artilleryTests/rosetta-load-test.yml:6]: `timeout: 60` should be under the `http:`, not `http: http:`

File: artilleryTests/rosetta-load-test.yml [hunk 2]
```diff
@@ -21,12 +23,13 @@ config:
       name: Spike phase
     - duration: 60
```
Comment (BerezinD) [inline artilleryTests/rosetta-load-test.yml:24]: This one is absolete

**Comment by BerezinD** [comment]
Good job!
But we still have 
`plugins.metrics-by-endpoint./block.codes.500: .................................. 1230`
Probably, because of the index.

File: artilleryTests/rosetta-load-test.yml (hunk 1 above)
Comment (MarMarMot) [reply artilleryTests/rosetta-load-test.yml:7]: Fixed

File: artilleryTests/rosetta-load-test.yml (hunk 2 above)
Comment (MarMarMot) [reply artilleryTests/rosetta-load-test.yml]: Fixed

**Review by Kammerlo — APPROVED** [review]

**Pull request merged**